tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +188,22 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_and_add_new_user(self):
+        response = self.client.get(reverse("admin:auth_group_add"))
+        self.assertContains(response, "<h1>%s</h1>" % (_("Add %s") % _("group")))
+        self.assertContains(
+            response,
+            '<label for="id_users">%s</label>' % _('Users')
+        )
```

Comment:
The title & scenario are a bit apart. The current implementation only checks that the page shows a widget for selecting users. However because no actual selection is made, nor the page is saved the tests don't check whether or not saving a group with a user is possible.